test/config-serializer.spec.js

```diff
@@ -29,9 +50,40 @@ describe('Config Serializer', () => {
     };
 
     let str = serializeConfig(cfg);
-    expect(str).to.be.equal(outCfg);
+    expect(str).toBe(outCfg);
+  });
+
+  it('can serialize System configuration', () => {
+    let cfg = readConfig([inpCfg]);
+    let outCfg =
+`System.config({
+  defaultJSExtensions: true
+});`;
+
+    let str = serializeConfig(cfg);
+    expect(str).toBe(outCfg);
   });
+  
+  it('can serialize System configuration', () => {
```

Comment:
I guess You meant "SystemJS" here instead of "System" (that is ok for the previous test)?